tfjs-layers/src/layers/convolutional_recurrent.ts

```diff
@@ -116,7 +116,7 @@ class ConvRNN2D extends RNN {
   /** @nocollapse */
   static className = 'ConvRNN2D';
 
-  readonly cell: ConvRNN2DCell;
+  declare readonly cell: ConvRNN2DCell;
```

Comment:
Nice catch here as well. `cell` is already a property of `RNN`, so `declare` is required here to assert its new type. I'm not completely sure why this didn't throw an error when compiling with Bazel.